### PR TITLE
Move down toolbar item when pressing the button

### PR DESCRIPTION
**Pull request by onurpolattimur**
Adds feature #1110

### CODE REVIEW

File: src/components/Toolbar.tsx [hunk 1]
```diff
@@ -55,7 +55,7 @@ const Toolbar = ({ dark, fontSize, toolbarOverlay, scrollPrioritized, showTopCon
   const [lastScrollLeft, setLastScrollLeft] = useState<number | undefined>()
   const [leftArrowElementClassName = 'hidden', setLeftArrowElementClassName] = useState<string | undefined>()
   const [rightArrowElementClassName = 'hidden', setRightArrowElementClassName] = useState<string | undefined>()
-
+  const [pressingToolbarId, setPressingToolbarId] = useState('')
```
Comment (raineorshine) [inline src/components/Toolbar.tsx:58]: Default state should be `null` which makes it more explicit. Empty string is more error prone.

```ts
useState<null | string>(null)
```

File: src/components/Toolbar.tsx [hunk 2]
```diff
@@ -195,12 +196,17 @@ const Toolbar = ({ dark, fontSize, toolbarOverlay, scrollPrioritized, showTopCon
                 <div
                   key={name}
                   id={id}
+                  style={pressingToolbarId === id ? { paddingTop: '10px' } : {}}
```
Comment (raineorshine) [inline src/components/Toolbar.tsx:199]: Let's add a `transition` so it's a bit more smooth.

**Review by raineorshine — CHANGES_REQUESTED** [review]

File: src/components/Toolbar.tsx [hunk 3]
```diff
@@ -195,12 +196,17 @@ const Toolbar = ({ dark, fontSize, toolbarOverlay, scrollPrioritized, showTopCon
                 <div
                   key={name}
                   id={id}
+                  style={pressingToolbarId === id ? { paddingTop: '10px', transition: 'padding 0.2s ease 0s' } : {}}
```
Comment (raineorshine) [inline src/components/Toolbar.tsx:199]: If `transition` is removed when `pressingToolbarId` is no longer equal to `id`, won't that remove the return animation? I think you want `transition` to always be set, and only remove `paddingTop`. Unless I am mistaken and it already transitions both directions even when `transition` is removed.

Comment (onurpolattimur) [reply src/components/Toolbar.tsx]: Yes, you are right! It will not apply animation for returning. I fixed that now.

**Review by raineorshine — APPROVED** [review]

**Pull request merged**